stylesheets/components/CompositionArea.scss

```diff
@@ -55,6 +55,7 @@
     align-items: center;
     background: none;
     border: none;
+    margin-bottom: 2px;
```

Comment:
Can I suggest an alternative change?
```suggestion
    width: 32px;
    height: 32px;
```

Reply:
So what your thoughts on this? :)

Reply:
Hi! Sorry for leaving this open, I assumed you would go with the original fix. I think your solution is better, in the codebase it appears that a lot of the buttons are set to 32x32 pixels in size as such, so it makes more sense to do it this way. I commited your suggestion, although I'm not sure if that was the right thing to do since it pollutes the commits. Either way I do hope this change gets done one way or another